tests/apps/test_morph_stats.py

```diff
@@ -233,12 +241,16 @@ def test_extract_dataframe_multiproc():
             for name in ['Neuron.swc', 'simple.swc']]
     with warnings.catch_warnings(record=True) as w:
         actual = ms.extract_dataframe(morphs, REF_CONFIG, n_workers=2)
+        # drop raw features as they require too much test data to mock
```

Comment:
@adrien-berchet please confirm whether you are ok with such approach. I use in in 4 places here.

Reply:
yes, it's fine, we don't care about the numbers, this is tested elsewhere

Reply:
It's ok for me too